text/0031-designated-operators.md

```diff
@@ -0,0 +1,51 @@
+# Designated node operators
+
+## Summary
+
+[summary]: #summary
+
+CEP PR: [casperlabs/ceps#0031](https://github.com/casperlabs/ceps/pull/0031)
+
+We propose that validators be given the capability to designate a public key different from their own to sign messages, thereby enabling validators to outsource their functions to third party operators.
+
+## Motivation
+
+[motivation]: #motivation
+
+It is expected that many validators will want to rely on third parties to provision and operate nodes. 
+
+## Guide-level explanation
+
+[guide-level-explanation]: #guide-level-explanation
+
+Add an additional field, expecting a public key (or some type wrapping a public key), to the `add_bid` entry point of the auction contract. The supplied public key will be used to sign consensus messages on the validator's behalf and to sign for or identify the validator in all instances required by the protocol proper. The link between the validator and the operator, as well as the history of validator's operator public keys, is to be tracked by the auction contract. Tracking of the pairings between a validator and its operators is intended to enable slashing, distribution of rewards and collection of transaction fees.
+
+## Reference-level explanation
+
+[reference-level-explanation]: #reference-level-explanation
+
+N/a
```

Comment:
```suggestion
N/A
```

But perhaps instead of fixing the typo, just delete the section?  I haven't been including them because it's just noise.

Reply:
Well, Not applicable rather than Not Applicable. I've included them just as a reminder for myself and readers to maintain discipline with these things (and invite people to critique the absence of content).